src/utils/subscriberBreaches.ts

```diff
@@ -101,13 +104,24 @@ export async function getSubBreaches(
         filterBreachDataTypes(breach.DataClasses);
       const resolvedDataClasses = (breachResolution[breach.Id]
         ?.resolutionsChecked ?? []) as ArrayOfDataClasses;
-      const dataClassesEffected = filteredBreachDataClasses.map((c) => {
-        if (c === BreachDataTypes.Email) {
-          return { [c]: [email.email] };
-        } else {
-          return { [c]: 1 };
-        }
-      });
+
+      const dataClassesEffected = filteredBreachDataClasses
+        .map((c) => {
+          // Exclude SSN breaches for non-US users as they are only relevant
+          // to US users as of now.
+          if (c === BreachDataTypes.SSN && countryCode !== "us") {
+            return null;
+          }
+          if (c === BreachDataTypes.Email) {
+            return { [c]: [email.email] };
+          } else {
+            return { [c]: 1 };
+          }
+        })
+        .filter(
+          (dataClassEffected) => dataClassEffected,
```

Comment:
this is just removing the `null` elements (in the case of L112-114 above where it's an SSN breach and the user isn't in the US), right?

Reply:
Nice, I extracted the function to its own file and replaced it here in [0ec5811](https://github.com/mozilla/blurts-server/pull/4283/commits/0ec58118c286aa19a7cdcf5e23cc5c26da3bd947).